### PR TITLE
Added terms and conditions

### PR DESCRIPTION
**Pull request by kj-crypto**

### CODE REVIEW

File: electrum/gui/qt/installwizard.py [hunk 1]
```diff
@@ -186,13 +186,14 @@ def on_change():
...
             self._set_gui_text()
-            self.config.set_key('language', language_abbreviation)
             self.refresh_gui()
 
         cb.currentIndexChanged.connect(on_change)
         # refresh config language
         on_change()
-        self.exec_layout(vbox, title=_('Select installation language'))
+        result = self.exec_layout(vbox, title=_('Select installation language'))
+        if result == 2:
```
Comment (PJLewandowski) [inline electrum/gui/qt/installwizard.py:195]: Could this be more descriptive?

File: electrum/gui/qt/terms_and_conditions_mixin.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        text_edit = TermsAndConditionsTextEdit(self.next_button)
+        text_edit.setText(text)
+        vbox.addWidget(text_edit)
+        self.next_button.setText(_('I agree'))
+        self.back_button.setText(_('I disagree'))
+        try:
+            # if user agreed with terms and conditions then result is 2
+            # otherwise GoBack exception is raised
+            result = self.exec_layout(vbox, title=_('Terms & Conditions'), next_enabled=True)
+            if result == 2:
```
Comment (PJLewandowski) [inline electrum/gui/qt/terms_and_conditions_mixin.py:107]: Could you make it more descriptive and remove above comment?

Comment (kj-crypto) [reply electrum/gui/qt/terms_and_conditions_mixin.py:107]: what about 
```python
# pushing 'I agree` button returns 2, 'I disagree` raises GoBack exception
pushed_button = self.exec_layout(vbox, title=_('Terms & Conditions'), next_enabled=True)
if pushed_button == 2:
```

File: electrum/gui/qt/installwizard.py (hunk 1 above)
Comment (kj-crypto) [reply electrum/gui/qt/installwizard.py:195]: what about
```python
# pushing `Next` button returns 2 and then config is modified
pushed_button = self.exec_layout(vbox, title=_('Select installation language'))
if pushed_button == 2:
```

**Review by PJLewandowski — APPROVED** [review]

**Pull request merged**